communication/resources/README.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Resources
+
+> A collection of resources organized by medium (e.g. audio, text, video)
+
+## Table of Contents
+
+<!-- vim-markdown-toc GFM -->
+
+- [Contributions](#contributions)
+- [Resources](#resources)
+  - [Audio](#audio)
+  - [Text](#text)
+  - [Video](#video)
+  - [Miscellaneous](#miscellaneous)
+
+<!-- vim-markdown-toc -->
+
+## Contributions
+
+If you would like to contribute to this list, please submit a PR and add `/sig contributor-experience` and `/assign @petermbenjamin`.
+
+The criteria for contributions are simple:
+
+- The resource must be related to Kubernetes.
+- The resource must be free.
+
+## Resources
+
+### Audio
+
+- [PodCTL](https://twitter.com/PodCTL)
+- [Kubernetes Podcast](https://kubernetespodcast.com)
+- [The New Stack Podcasts](https://thenewstack.io/podcasts/)
+- [SoundCloud](https://soundcloud.com/search?q=kubernetes)
+
+### Text
+
+- [CNCF Blog](https://www.cncf.io/newsroom/blog/)
+- [Dev.To](https://dev.to/t/kubernetes)
+- [Heptio Blog](https://blog.heptio.com)
+- [KubeTips](http://kubetips.com)
+- [KubeWeekly](https://twitter.com/kubeweekly)
+- [Kubedex](https://kubedex.com/category/blog/)
+- [Kubernetes Blog](https://kubernetes.io/blog/)
+- [Kubernetes Changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md)
```

Comment:
I would omit. Inner-project resources should probably be referenced elsewhere to keep the spirit of this list.

Reply:
Will omit.